zcncore/transaction.go

```diff
@@ -1787,3 +1806,27 @@ func (t *Transaction) WritePoolUnlock(poolID string, fee int64) (
 	go func() { t.submitTxn() }()
 	return
 }
+
+//
+// ZCNSC transactions
+//
+
+func (t *Transaction) ZCNSCUpdateGlobalConfig(ip *InputMap) (err error) {
+	err = t.createSmartContractTxn(ZCNSCSmartContractAddress, transaction.ZCNSC_UPDATE_GLOBAL_CONFIG, ip, 0)
+	if err != nil {
+		Logger.Error(err)
+		return
+	}
+	go func() { t.submitTxn() }()
```

Comment:
no need to run it in a funciton, please use go t.submitTxn()

Reply:
done